distribution/docs/src/main/resources/_contents/_solr-contents/managing-solr-contents.adoc

```diff
@@ -85,9 +53,16 @@ Before the ${ddf-solr} Application can be installed:
 
 ===== Installing
 
-. Navigate to the ${ddf-solr} application in the Admin console.
-. Under the *Features* tab, uninstall the `catalog-solr-external-provider` feature.
-. Install the `catalog-solr-embedded-provider`
+. Edit the <INSTALLATION_DIR>/etc/system.properties
+
+.. Comment out the Solr Client Configuration for *Cloud Solr Client* and *Embedded Solr Client*.
+.. Uncomment the section for the *Http Solr Client*:
```

Comment:
Are these really the steps for the Embedded Solr Client?  It looks like they are for the Http Solr Client.